c2/file-transfer/write-and-execute-a-file.yml

```diff
@@ -4,8 +4,6 @@ rule:
     namespace: c2/file-transfer
     author: moritz.raabe@fireeye.com
     scope: function
-    att&ck:
-      - Command and Control::Ingress Tool Transfer [T1105]
```

Comment:
we have this under C2, but are we sure that this rule encodes command and control?

Reply:
I don't think it's possible to know it's C2... I think something similar can be said for many rules/mappings, but I think there is value in suggesting such objectives/behaviors as possibilities (might be useful to associate confidence values with mappings). Could replace the current att&ck and mbc mapping with mbc micro behaviors (or add for context), which are literal/certain -- File System::Write File [C0052] and Process::Create Process [C0017].